openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyValue.tsx

```diff
@@ -1094,14 +1103,15 @@ export const PropertyValue: FC<PropertyValueProps> = ({
             {showInput ? getPropertyInput() : getValueElement()}
           </div>
           {isOverflowing && !showInput && (
-            <Button
-              className="custom-property-value-toggle-btn"
+            <Icon
+              className={classNames('custom-property-value-toggle-btn', {
+                active: isExpanded,
+              })}
+              component={ArrowIconComponent}
               data-testid={`toggle-${propertyName}`}
-              size="small"
-              type="text"
-              onClick={toggleExpand}>
-              {isExpanded ? <UpOutlined /> : <DownOutlined />}
-            </Button>
+              style={{ color: DE_ACTIVE_COLOR, ...ICON_DIMENSION }}
+              onClick={toggleExpand}
+            />
```

Comment:
We should not be doing this. Any reason why we are assigning the onClick to the icon directly and removing the button?